Plugins/Flow.Launcher.Plugin.WebSearch/Languages/ru.xaml

```diff
@@ -29,7 +29,8 @@
         Thus, the generic formula for a search on Netflix is https://www.netflix.com/search?q={q}
     </system:String>
 
-
+    <system:String x:Key="flowlauncher_plugin_websearch_copyurl_title">Скопировать URL-адрес</system:String>

```

Comment:
Translation are done via Crowdin, while this works for now, when next Crowdin translation push happens this will revert to English. Best is go to Crowdin to add this in.

Reply:
kay, will do some Crowdin things when get time